QMWP.php

```diff
@@ -1791,6 +1797,16 @@ function add_shortcake_to_search_correlations()
                             'outcomes' => 'Outcomes',
                         ),
                     ),
+
+                    array(
+                        'label' => 'Common or user measurements',
```

Comment:
Should this say "Specify whether to show common aggregated correlations or the user's own correlations." ?